core/modules/util/threadSafe.h

```diff
@@ -74,17 +74,20 @@ class Flag {
     /** Sets flag value to 'val' and returns the old value of flag.
      */
     virtual T set(T val) {
```

Comment:
General comment - we should probably replace this with `std::atomic<T>`. Definitely not for this ticket.